doc/sailfish-webview-webview.qdoc

```diff
@@ -14,4 +14,60 @@
   import namespace.
 */
 
+/*!
+  \qmlproperty var WebView::popupComponents
+  \brief Contains a dictionary of custom user interaction components
+
+  Client applications can define their own custom user interaction components
+  which extend the various interface types defined in the
+  \l{Sailfish.WebView.Popups} namespace.
+
+  In order for those custom components to be used by the WebView, the client
+  must then register those components with the WebView via this property.
+  The value is a dictionary whose keys are the asynchronous message types
+  which trigger the popup, and whose values are the \c{Component}s (or
+  component type definition URLs) which the WebView should load and display
+  when it receives the trigger message.
```

Comment:
We should expose on the API level those internal messages directly. Rather we should have internal lookup map to map them. I'd suggest enum as key and then map that custom popup type to respected string.